projects/wave-core-new/src/lib/tabs/tabs.component.scss

```diff
@@ -37,7 +37,6 @@
 ::ng-deep .mat-tab-header {
     height: 3rem;
 
-    margin-left: 3rem;
     @media (max-width: 600px) and (orientation: portrait) {
```

Comment:
Why did you remove this?

Reply:
because it made the tabs in the upload dialog look weird and I wasn't able to overwrite this setting. As I don't see where else it is used I removed it.

Reply:
:host ::ng-deep …

Reply:
45ca50031043e21f611db6f3ae861c14f7f756ab